### DISCUSSION TITLE
Videos on YouTube

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#340] [Ideas]
Saw your post on LinkedIn and had a look at the videos 👍.   I thinks they should be a referenced from the help.  And maybe also from a walkthrough.  Actually, probably put them in the help and link from the walkthrough to the help section, so you can add or changes videos in only one place.

I'm not an expert on YouTube, but maybe it is possible to create a "Code for IBM i" channel?

Thoughts?

(I'm not a great fan of the documentation videos the the VSCODE folk produce because generally they fail to mention which keys are being pressed, but yours are much better.)

**Comment by worksofliam** [comment]
All good ideas.

I actually added the videos to the repo readme for the time being.

> I'm not an expert on YouTube, but maybe it is possible to create a "Code for IBM i" channel? 

A cool idea.. but something I don't have a bunch of time for. If someone else would like to, that'd be cool.

**Reply by SJLennon** [reply]
Putting the videos in the readme is cool..
